docs/www/advocacy/white_camel/index.html

```diff
@@ -14,12 +14,13 @@
     The awards were initiated by <a href="http://www.pm.org/">Perl Mongers</a> and <a href="http://oreilly.com">O'Reilly &amp; Associates</a> at the 1999 Perl Conference. Today, <a href="http://www.perlfoundation.org">The Perl Foundation</a> acknowledges these exceptional individuals annually at the O'Reilly Open Source Conference.
 </p>
 <p>
-    The <a href="2014.html">2014 winners</a> have been <a href="http://blogs.perl.org/users/brian_d_foy/2014/12/the-2014-white-camel-awards.html">announced</a>.
+    The <a href="2014.html">2015 winners</a> have been <a href="http://blogs.perl.org/users/brian_d_foy/2015/12/the-white-camel-awards-2015-edition.html">announced</a>.
```

Comment:
I've fixed in https://github.com/perlorg/perlweb/commit/604b9550006c9464eaefa8198ca698fd494dd174

I've merged will need @abh or @rspier to deploy though